apps/visualization/management/commands/sync_filter_options.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand
+from apps.visualization.models import DataCountryLevel
+from apps.migrate_csv.models import CountryFilterOptions
+
+
+class Command(BaseCommand):
+    help = 'Command to sync distinct filter options'
+
+    def handle(self, *args, **options):
+
+        # Remove old filter options
+        CountryFilterOptions.objects.all().delete()
```

Comment:
We can skip this. and use conflict feature in bulk_create

https://docs.djangoproject.com/en/4.1/ref/models/querysets/#bulk-create

Eg: https://github.com/mapswipe/python-mapswipe-workers/blob/project/group-analytics/django/apps/aggregated/management/commands/import_aggregated_data.py#L44-L54